repository/domains/headers.go

```diff
@@ -0,0 +1,95 @@
+package domains
```

Comment:
IMHO there is no need to put it in a separate package, it could be part of the repository package.

Reply:
Can not move this to repository package because of import cycle with `/data/sql/headers.go`

Reply:
So maybe just rename it to model or something that doesn't mislead that it's part of the domain logic?

Reply:
Ok I renamed it to `dto`